src/Content/Text/Plaintext.php

```diff
@@ -299,10 +305,13 @@ private static function getParts(string $message, int $limit): array
 			if (strlen($part . $word) > ($limit - 8)) {
 				$parts[] = trim($part);
 				$part = '';
+				if (strlen($message) <= ($limit - 8)) {
+					$limit -= 23;
```

Comment:
Where is this `23` value coming from? Please indicate its source in a comment.

Reply:
I replaced it with a constant.